### ISSUE TITLE
How to render local PDF

### ISSUE THREAD

**Issue by kirtishjain** [rumax/react-native-PDFView#42]
Hi,

Thanks alot for your library but I've an issue while using it.

How could I render my PDF file which is present in one of my folder (images) of project itself. e.g.

images
I->'xyz.pdf'

in a file which is present at the same level of images directory.

I tried your code in following manner:

```
<View style={{ flex: 1 }}>
              <PDFView
                style={{ flex: 1 }}
                onError={(error) => console.log('onError', error)}
                onLoad={() => console.log('onLoad')}
                resource="./images/xyz.pdf"
                resourceType= "file"
              />
            </View>
```

Buut not able to access my PDF.

Please help me out to access my local PDF file. It's quite urgent.

Thanks in advance.

**Comment by sanderfrenken** [comment]
Hi there,

Currently we don't support loading PDF's from local filesystem. 
We would like to add it though, but can't promise you anything on when this will be implemented.

We will let you know in this issue when we have an update on this topic

**Comment by sanderfrenken** [comment]
Hi again,

Just implemented local file loading for iOS.
It is published to npm (0.3.0). 
It loads from mainBundle, if it fails there to find the document it will go to the documentsDir.

Please let us know if this works for now.
Android is not yet available, but will be working on it in the near future.

**Comment by rumax** [comment]
@kirtishjain version 0.3.1 supports local files now for both android and ios. For more details about usage, please, check readme and demo project. 
Also do not forget to star the project if you like it.
